src/lib/spinel/spinel.h

```diff
@@ -377,7 +377,7 @@
  * Please see section "Spinel definition compatibility guideline" for more details.
  *
  */
-#define SPINEL_RCP_API_VERSION 6
+#define SPINEL_RCP_API_VERSION 7
```

Comment:
We only change `SPINEL_PROP_THREAD_PARENT`. This property is only available/used on FTD/MTD builds and not used on RCP. So we don't need to increment the `RCP_API_VERSION` (we increment it only if we have a change to a property used in RCP build).